Automattic/jetpack
# VideoPress: enqueue token bridge script in the block editor context

Fixes https://github.com/Automattic/jetpack/issues/27020

#### Changes proposed in this Pull Request:

* VideoPress: enqueue token bridge script when `init` action

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Create a video VideoPress video
* Set privacy -> private
* Go to block editor
* Create a new VideoPress Video v6 block
* Use the private video
* Confirm, without these changes, the video is not visualized, getting the `Post message time out`
* Confirm with these changes the video is visualized

before | after
------|------
<img src="https://user-images.githubusercontent.com/77539/197423161-155b55d7-a09e-4589-8938-13f40919e05b.png" width="400px" /> | <img width="400px" alt="image" src="https://user-images.githubusercontent.com/77539/197423172-399abac8-8eb9-42fd-b079-190a2360ef0b.png">
